R/ae_attendances_admissions_age_sex.R

```diff
@@ -29,6 +29,7 @@ ae_attendances_admissions_age_sex <- function(start_date, end_date, data, plot_c
     dplyr::filter(spell_start <= end_date & spell_end >= start_date) %>%
     dplyr::select(spell_number, spell_start, spell_end, gender, age_band_start, spell_class_col)
 
+  dt$age_band_start <- factor(dt$age_band_start, levels = gtools::mixedsort(unique(dt$age_band_start)))
```

Comment:
This looks great - if possible, could we use `dplyr::mutate` here? So
`dt <- dt %>% dplyr::mutate(age_band_start = factor(age_band_start, levels = ...as your code...))`

Reply:
Yes, you should see the code rewritten with `dplyr` [here](https://github.com/HorridTom/hospitalflow/pull/105/commits/a4b136ccba8a5a77af5eb3bb18fd05863be6d1bf). I tried to put `dplyr::mutate(...` in line 31 and `%>%` at the end of line 30 (it makes the code more concise and avoids `dt <- dt %>%` line), but I got a strange error about closure being not subsettable, so left it like this.